pkg/prompts/prompts.go

```diff
@@ -244,3 +300,37 @@ func Select[T any](label string, items []T, opt *SelectOpt[T]) (T, error) {
 
 	return items[i], nil
 }
+
+func getCurrentDirName() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		return "", fmt.Errorf("getting current directory: %v", err)
+	}
+	dirName := filepath.Base(dir)
+	return sanitizeAppName(dirName), nil
+}
+
+// Sanitize the directory name to comply with k8s label rules
+func sanitizeAppName(name string) string {
+	var builder strings.Builder
+
+	// Remove all characters except alphanumeric, '-', '_', '.'
+	for _, r := range name {
+		if unicode.IsLetter(r) || unicode.IsDigit(r) || r == '-' || r == '_' || r == '.' {
+			builder.WriteRune(r)
+		}
+	}
+
+	sanitized := builder.String()
+	if sanitized == "" {
+		sanitized = defaultAppName
+	} else {
+		// Trim leading and trailing '-', '_', '.'
+		sanitized = strings.Trim(sanitized, "-._")
+		// Ensure the length does not exceed 63 characters
+		if len(sanitized) > 63 {
```

Comment:
I'm thinking Trim() should be called after the if statement here. If the string is longer than 63 characters, then Trim() would act upon that last char but ultimately that last char would get cut out by the if statement and that "new" last char wouldn't have been checked.

Reply:
yeah, you are right! that's a great catch!